### PR TITLE
Story package iii

### PR DESCRIPTION
**Pull request by blongden73**
## What does this change?
Fixes the comment count on the story package, and gets the pillar colour into the quote mark. 

Tightens up some of the spacing. 

picture 1019

### CODE REVIEW

File: static/src/stylesheets/inline/story-package-garnett.scss [hunk 1]
```diff
@@ -17,9 +21,19 @@
     }
 }
 
+#headlines.fc-container {
```
Comment (SiAdcock) [inline static/src/stylesheets/inline/story-package-garnett.scss]: Is there a way of targetting this element without using the `id`? The specificity of this selector looks a bit heavy

File: static/src/stylesheets/inline/story-package-garnett.scss [hunk 2]
```diff
@@ -17,9 +21,19 @@
...
+        display: none;
+    }
+}
+
 .fc-container--story-package {
     background-color: $garnett-neutral-4;
 
+    .fc-container__toggle {
+        display: none;
+    }
```
Comment (SiAdcock) [inline static/src/stylesheets/inline/story-package-garnett.scss]: Is this necessary, given lines 25-27?

**Comment by paperboyo** [comment]
@blongden73 This is not Garnett-related question
, because this was always broken
: is there a way to keep opening quotes always on the same line as subsequent quoted text
? Because
, currently
, we have “
funny
-looking
” typography
, if one treats these quotes
… as quotes and not just some graphics
?

**Review by zeftilldeath — APPROVED** [review]
🍭 🤙

**Pull request merged**